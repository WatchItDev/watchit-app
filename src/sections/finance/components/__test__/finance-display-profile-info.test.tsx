import { describe, it } from "vitest";
import { render } from "@testing-library/react";
import FinanceDisplayProfileInfo from "../finance-display-profile-info";
import { Profile } from "@lens-protocol/api-bindings";


const carouselMock = {
  currentIndex: 0,
  nav: undefined,
  carouselSettings: {},
  carouselRef: { current: null },
  onPrev: () => {},
  onNext: () => {},
  onSetNav: () => {},
  onTogo: () => {},
  setNav: () => {},
  setCurrentIndex: () => {},
};
const initialListMock: Profile[] = [];

const renderComponent = () => {
  return render(
    <FinanceDisplayProfileInfo
      initialList={initialListMock}
      carousel={carouselMock}
      mode="profile"
    />,
  );
};
describe("[COPONENTS]: <FinanceDisplayProfileInfo />", () => {
  it("to match snapshot", () => {
    const { container } = renderComponent();
    expect(container).toMatchSnapshot();
  });
});
