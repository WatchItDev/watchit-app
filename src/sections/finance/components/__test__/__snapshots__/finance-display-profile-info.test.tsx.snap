// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[COPONENTS]: <FinanceDisplayProfileInfo /> > to match snapshot 1`] = `<div />`;
